packages/docs/components/button.md

```diff
@@ -128,103 +110,243 @@ It’s best to use this button on a white background.
 
 <Description>
 
-These buttons may only be used on top of a “Base Color” such as Purple 500 or on top of a colored background, photo, or illustration. They should only be used in the scenario where Primary, Secondary, and regular text links cannot be used.
+Use Overlay Buttons when Primary or Secondary variants aren't visually suitable. These Buttons may only be used on top of a base-variant background color, a photo, or an illustration.
 
-Lastly, they only exist at the large size.
+</Description>
+
+<Visual>
+  <template>
+    <button class="ods-button is-ods-button-overlay">Register for Spacecamp</button>
+  </template>
+</Visual>
+
+## States
+
+<Description>
+
+Odyssey provides visual affordances for all standard button states.
+
+</Description>
+
+### Enabled
+
+<Description>
+
+Buttons in their "normal" state are considered enabled. They are ready for user interaction.
 
 </Description>
 
 <Visual>
   <template>
-    <button class="ods-button is-ods-button-overlay">Default</button>
-    <button class="ods-button is-ods-button-overlay is-ods-button-hover">Hover</button>
-    <button class="ods-button is-ods-button-overlay is-ods-button-focus">Focus</button>
-    <button disabled="disabled" class="ods-button is-ods-button-overlay">Disabled</button>
+    <button class="ods-button">Launch</button>
+  </template>
+</Visual>
+
+### Focus
+
+<Description>
+
+The focus state is a visual affordance that the user has highlighted the Button with a pointer, keyboard, or voice.
+
+</Description>
+
+<Visual>
+  <template>
+    <button class="ods-button is-ods-button-focus">Launch</button>
   </template>
 </Visual>
 
-## Guidelines
+### Hover
 
 <Description>
 
-Use buttons to indicate the important actions that a user can take. Don’t use buttons to navigate around the site or product; use links instead
+Hover states are activated when the user pauses their pointer over the Button.
 
 </Description>
 
+<Visual>
+  <template>
+    <button class="ods-button is-ods-button-hover">Launch</button>
+  </template>
+</Visual>
+
+### Disabled
+
+<Description>
+
+Disabled Buttons are unavailable for interaction and cannot be focused.
```

Comment:
This is sort of a given. Consider rewording to something like

> Disabled Buttons can be used when the user is not allowed to proceed due either to incomplete information or access restrictions

Reply:
Good call out for a guideline. I may combine there. It's surprising to me, but we've had multiple people ask why x/y/z didn't happen when a Button was disabled (including users not being able to click it).